genie-common/src/main/java/com/netflix/genie/common/dto/JobStatusMessages.java

```diff
@@ -115,6 +115,78 @@
     public static final String JOB_SETUP_FAILED =
         "Job script failed during setup. See setup log for details";
 
+    /**
+     * Agent failed to claim job id, job record does not exist, it is not in a state where it can be claimed.
+     */
+    public static final String FAILED_TO_CLAIM_JOB = "Failed to claim job";
```

Comment:
This file is still using the old dto locations and the old `genie-common` package. We might want to move them somewhere better like `genie-common-external` as I'd like to get rid of `genie-common` eventually